workspaces/analytics/plugins/analytics-provider-segment/package.json

```diff
@@ -61,7 +61,7 @@
     "@testing-library/react": "14.3.1",
     "@testing-library/user-event": "14.5.2",
     "@types/node": "18.19.34",
-    "@types/react": "18.3.3",
+    "@types/react": "18.3.12",
```

Comment:
can we do the same with other dependencies so that we don't get bumps so often @jmezach?

```suggestion
    "@types/react": "^18",
```

Reply:
He only owns the New Relic analytics plugin. @04kash can you make this change?

Reply:
Yup, taking a look

Reply:
Based on what was discussed during the SIG this week, the devDependencies have been pinned following renovate guidelines. So I assume we should keep them pinned for now?